C/Cpp_include/c4BlobStore.hh

```diff
@@ -86,43 +85,48 @@ namespace C4Blob {
     bool findBlobReferences(FLDict C4NULLABLE, const FindBlobCallback&);
 
     /** Finds old-style attachment references, i.e. sub-dictionaries of "_attachments". */
-    bool findAttachmentReferences(FLDict C4NULLABLE docRoot, const FindBlobCallback &callback);
-};
-
+    bool findAttachmentReferences(FLDict C4NULLABLE docRoot, const FindBlobCallback& callback);
+};  // namespace C4Blob
 
-struct C4ReadStream : public fleece::InstanceCounted, C4Base {
+struct C4ReadStream
+    : public fleece::InstanceCounted
+    , C4Base {
     C4ReadStream(const C4BlobStore&, C4BlobKey);
     C4ReadStream(C4ReadStream&&);
     ~C4ReadStream();
-    size_t read(void *buffer, size_t maxBytes);
+    size_t  read(void* buffer, size_t maxBytes);
```

Comment:
What is the reasoning for this one being different than the others with the `*` placement?

Reply:
That's very strange. I have no idea. Possibly some of the settings I chose

Reply:
Turns out `DerivePointerAlignment` was on. This means Clang-Format will analyze each file's most common pointer placement, and use that for that file. In this file, most pointers and references were attached to the type, so it followed that.